src/actions/accounts.js

```diff
@@ -11,10 +11,12 @@ const removeAccountFromGroup = (group, account) => {
 }
 
 const getStackCollection = doctype => {
+  const links = getLinks()
   return links.stack.client.collection(doctype)
 }
 
 const deleteOrphanOperations = async (client, account) => {
+  const links = getLinks()
```

Comment:
I don't understand why we always want to use the stack link here. We can't use the pouch synchronization for this ?

Reply:
Maybe we could but here the main thing was to fix the wrongly used links.